reference/halyard/high-availability.md

```diff
@@ -103,7 +103,7 @@ hal config deploy ha echo enable
 When Spinnaker is deployed with this enabled, Echo will be deploy as two different services:
 
 * [`echo-scheduler`](#echo-scheduler)
-* [`echo-slave`](#echo-slave)
+* [`echo-replica`](#echo-replica)
 
 Although the logical Echo services still cannot be horizontally scaled, splitting the services will reduce the load on both.
```

Comment:
One of them can be horizontally scaled, right?

Reply:
Thanks for catching that. I'll make an update to this now.

Reply:
Fixed